qa-pipelines/tasks/run-test.sh

```diff
@@ -74,18 +75,38 @@ EOF
 }
 
 container_status() {
-  kubectl get --output=json --namespace=scf pod $1 \
+  kubectl get --output=json --namespace="${CF_NAMESPACE}" pod $1 \
     | jq '.status.containerStatuses[0].state.terminated.exitCode | tonumber' 2>/dev/null
 }
 
 image=$(awk '$1 == "image:" { gsub(/"/, "", $2); print $2 }' "${CAP_DIRECTORY}/kube/cf${CAP_CHART}/bosh-task/${TEST_NAME}.yaml")
 
+test_pod_yml="${CAP_DIRECTORY}/kube/cf${CAP_CHART}/bosh-task/${TEST_NAME}.yaml"
+test_non_pods_yml=
```

Comment:
why this, if its defined below?

Reply:
Because nounset throws an error for the other tests if it's not set